### PR TITLE
Need review this code

### PR DESCRIPTION
**Pull request by jimanisothername**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Alur Fitur Login:
 2. Memvalidasi masukan.
 3. Menampilkan halaman home.
 
-**Line for squashing 1**
-**Line for squashing 2**
-**Line for squashing 3**
+**Line for squashing 1**
```
Comment (fidojahfal) [inline README.md:10]: What is the difference between this code and before?

**Review by fidojahfal — COMMENTED** [review]
Need explain what is the difference between the code

File: README.md (hunk 1 above)
Comment (jimanisothername) [reply README.md:10]: The difference is every end of the line, I add two spaces like this "  " to add new line.

**Review by fidojahfal — APPROVED** [review]
OK. Approved!

**Pull request merged**